client/state/products-list/schema.js

```diff
@@ -0,0 +1,33 @@
+export const productsListSchema = {
+	type: 'object',
+	required: [],
+	patternProperties: {
+		'.+': {
```

Comment:
Hmm, a pattern like the below should work, but there are some interesting key names (like `no-adverts/no-adverts.php`) so I wasn't sure if we should constrain it to the current list?

`^[A-z0-9_-\.\/\\]+$`